core/src/main/java/org/elasticsearch/transport/netty/NettyTransportChannel.java

```diff
@@ -134,6 +141,14 @@ public void sendResponse(Throwable error) throws IOException {
         transportServiceAdapter.onResponseSent(requestId, action, error);
     }
 
+    private void close() {
```

Comment:
@danielmitterdorfer Why is this method called `close`? It might make more sense if it were in the `finally` blocks but as-is I'm confused by it. Help?